### PR TITLE
First create PiNozCam.md

### PR DESCRIPTION
**Pull request by DrAlexLiu**

### CODE REVIEW

**Comment by DrAlexLiu** [comment]
First create the PiNozCam and publish

File: _plugins/PiNozCam.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+layout: plugin
+
+id: PiNozCam
```
Comment (jneilliii) [inline _plugins/PiNozCam.md]: should be `pinozcam` to match setup.py

**Review by jneilliii — CHANGES_REQUESTED** [review]
Is there a recommended pi version for handling the AI image processing? Might be a good idea to include that here in the plugin listing. 

There are a couple of issues. [here](https://github.com/DrAlexLiu/OctoPrint-PiNozCam/blob/f686a58c0c09c2c7506486e4eb41711e5aadbc5c/octoprint_pinozcam/__init__.py#L436) needs to match your plugins identifier from setup.py. Change `updateplugindemo` to `pinozcam`. Same is true for the id in your plugin listing md file, as well as the filename itself. `PiNozCam.md` should be named `pinozcam.md`.

**Comment by jneilliii** [comment]
Couple of other house-keeping items. Delete the folders [OctoPrint_PiNozCam.egg-info](https://github.com/DrAlexLiu/OctoPrint-PiNozCam/tree/f686a58c0c09c2c7506486e4eb41711e5aadbc5c/OctoPrint_PiNozCam.egg-info) and [__pycache__](https://github.com/DrAlexLiu/OctoPrint-PiNozCam/tree/f686a58c0c09c2c7506486e4eb41711e5aadbc5c/octoprint_pinozcam/__pycache__), they shouldn't be included in your plugin's repo and are typically added to .gitignore to prevent them from being uploaded.

**Comment by jneilliii** [comment]
you may also want to include a screenshot of the tab, it's highly recommended.

**Review by jneilliii — APPROVED** [review]

**Comment by DrAlexLiu** [comment]
Thanks for approval.

**Pull request merged**